Sources/ClientRuntime/Middleware/RetryMiddleware.swift

```diff
@@ -66,3 +102,24 @@ public struct RetryMiddleware<Strategy: RetryStrategy,
         }
     }
 }
+
+// Calculates & returns TTL datetime in strftime format `YYYYmmddTHHMMSSZ`.
+func getTTL(now: Date, estimatedSkew: TimeInterval, socketTimeout: TimeInterval) -> String {
+    let dateFormatter = DateFormatter()
+    dateFormatter.dateFormat = "yyyyMMdd'T'HHmmss'Z'"
+    dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+    dateFormatter.timeZone = TimeZone(abbreviation: "GMT")
```

Comment:
nit: use TZ identifier "UTC" for server communications

Reply:
Changed to UTC.